sdk/storage/storage-file-share/README.md

```diff
@@ -114,31 +105,38 @@ For example, you can create following CORS settings for debugging. But please cu
 
 ## Examples
 
-### Import types
+### Import the package
+
+Tu use the clients, import the package into your file:
 
-You can use the `const Azure = require("@azure/storage-file-share");` shown above then use types and functions from `Azure`.
-Or you can selectively import certain types,
+```javascript
+const AzureStorageFileShare = require("@azure/storage-file-share");
+```
+
+Alternative, selectively import only the types you need:
 
 ```javascript
-const { ShareServiceClient, SharedKeyCredential } = require("@azure/storage-file-share");
+const { ShareServiceClient, StorageSharedKeyCredential } = require("@azure/storage-file-share");
 ```
 
-### Create the file service client
+### Create the share service client
 
-Use the constructor to create a instance of `ShareServiceClient`, passing in the credential.
+Use the constructor to create a instance of `ShareServiceClient`. It needs to authenticate with the Azure service, so pass in a `StorageSharedKeyCredential` with your account and key.
 
 ```javascript
+const { ShareServiceClient, StorageSharedKeyCredential } = require("@azure/storage-file-share");
```

Comment:
Please double check all the samples in README.md can be run properly.

Reply:
I validated them all with a storage account through my Visual Studio Enterprise subscription.